include/dts-functions.sh

```diff
@@ -1387,3 +1387,92 @@ show_menu() {
     echo -ne "${RED}${SSH_OPT_UP}${NORMAL} to launch SSH server  ${NORMAL}"
   fi
 }
+
+check_if_fused() {
+  file_path="/sys/class/mei/mei0/fw_status"
+
+  if [[ ! -f $file_path ]]; then
+      echo "File not found: $file_path"
+      return 2
+  fi
+
+  hfsts6_value=""
+  line_number=1
+  while IFS= read -r line; do
+      if [[ $line_number -eq 6 ]]; then
+          hfsts6_value=$line
+          break
+      fi
+      ((line_number++))
+  done < "$file_path"
+
+  if [[ -z $hfsts6_value ]]; then
+      echo "Failed to read HFSTS6 value"
+      exit 1
```

Comment:
This indent has 6 spaces instead of 4 as in other places in this script.

Reply:
We could use `shfmt -i 4 include/dts-functions.sh` because shellcheck doesn't check indents.
There is also pre-commit available https://github.com/scop/pre-commit-shfmt
As this repo contains only shell scripts I think it could be very useful, though there would be a lot of fixes applied at first:

```diff
diff --git a/.pre-commit-config.yaml b/.pre-commit-config.yaml
index 3e92a36fc7e2..4019e6f3e9fe 100644
--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -45,6 +45,12 @@ repos:
       - id: markdownlint
       - id: markdownlint-fix

+  - repo: https://github.com/scop/pre-commit-shfmt
+    rev: v3.9.0-1
+    hooks:
+      - id: shfmt
+        args: ["-i", "4"]
+
   - repo: https://github.com/koalaman/shellcheck-precommit
     rev: v0.9.0
     hooks:
```

Not sure if we want to add this tool?

Reply:
0548c9b9045e4fb30addeb8645a0c353d11c8082

Reply:
0548c9b9045e4fb30addeb8645a0c353d11c8082